pkg/manager/member/tikv_group_member_manager.go

```diff
@@ -92,3 +145,537 @@ func (tgm *tikvGroupMemberManager) registerTiKVGroup(tg *v1alpha1.TiKVGroup, tc
 	msg := fmt.Sprintf("tg[%s/%s] register itself to tc[%s/%s] successfully, requeue", tg.Namespace, tg.Name, tcNamespace, tcName)
 	return controller.RequeueErrorf(msg)
 }
+
+func (tgm *tikvGroupMemberManager) syncServiceForTiKVGroup(tg *v1alpha1.TiKVGroup) error {
+	//TODO: support Pause
+
+	ns := tg.Namespace
+	svcName := controller.TiKVGroupPeerMemberName(tg.Name)
+
+	newSvc := newServiceForTiKVGroup(tg, svcName)
+	oldSvcTmp, err := tgm.svcLister.Services(ns).Get(svcName)
+	if errors.IsNotFound(err) {
+		err = controller.SetServiceLastAppliedConfigAnnotation(newSvc)
```

Comment:
What is the purpose of SetServiceLastAppliedConfigAnnotation, can you add some comments for it

Reply:
currently, most k8s native api we used have been added the `LastAppliedConfigAnnotation`, we use this to compare the equality between 2 different k8s api in one kind. @aylei  Could you provide more detail why using this strategy?